gardener-addon/files/certificate.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+echo "---> Get Shoot Domain"
+SHOOT_DOMAIN="$(kubectl -n kube-system get configmap shoot-info -o jsonpath='{.data.domain}')"
+DOMAIN="$SUBDOMAIN.$SHOOT_DOMAIN"
+
+echo "---> Requesting certificate for domain ${DOMAIN}"
```

Comment:
Please be consistent in the case of code style. Personally I prefer `${DOMAIN}` instead of `$DOMAIN`.

So in general, use env with `{}` or without, but please be consistent :)